frontend/public/scss/home-page/home-page-contact-row.scss

```diff
@@ -13,6 +13,10 @@ body.new-design {
   @include media-breakpoint-down(lg) {
     background: $home-page-grey-lite;
   }
```

Comment:
Is the gradient above required?  It looks like the background will always be a solid `$home-page-border-grey` regardless of screen size.

Reply:
Yes, it's what's creating that line down the middle :)